app/src/test/java/com/example/memeapp/domain/usecase/GetSingleMemeUseCaseTest.kt

```diff
@@ -12,17 +14,33 @@ import org.junit.Test
 
 class GetSingleMemeUseCaseTest {
     private var memesService: MemesService = mock()
-    private var meme: MemesEntity = mock()
+    private var memeLocalDatabase: LocalMemeDatabase = mock()
+    private var meme: Observable<MemesEntity> = mock()
+    private var memes: MemesEntity = mock()
     private lateinit var getSingleMemeUseCaseImpl: GetSingleMemeUseCaseImpl
 
     @Before
     fun setUp() {
-        getSingleMemeUseCaseImpl = GetSingleMemeUseCaseImpl(memesService)
+        getSingleMemeUseCaseImpl = GetSingleMemeUseCaseImpl(memesService, memeLocalDatabase)
     }
 
     @Test
-    fun call() {
-        whenever(memesService.getSingleMeme(ID)).thenReturn(Observable.just(meme))
+    fun `test local data base`(){
+        whenever(memeLocalDatabase.getSingleMeme(ID)).thenReturn(memes)
```

Comment:
you are using this line in all three test.

when this happen, you can remove line from tests, and put it in de @Before setUp() block